src/lib/rest/rest.cpp

```diff
@@ -604,7 +604,7 @@ static int contentTypeCheck(ConnectionInfo* ciP)
 
```

Comment:
Ah, of course!
When the branch was prepared (all these queued PR-branches), develop wasn't ready and I couldn't touch CHANGES_NEXT_RELEASE. I had completely forgotten ...